isis/src/base/objs/Table/Table.cpp

```diff
@@ -137,6 +138,60 @@ namespace Isis {
     }
   }
 
+  /**
+   * This constructor takes in a string to create a Table object.
+   * 
+   * @param tableName The name o the Table to be read
+   * @param tableStr The table string
+   * @param fieldDelimiter The delimiter to separate fields with
+  */
+  Table::Table(const QString &tableName, const std::string &tableString, const char &fieldDelimiter) {
```

Comment:
I think if we are reading in Jigsaw results we should be able to use the ISIS CSVReader class instead of implementing the reader in Table. I could be missing something about our CSV Reader though.

Something to try when we get back to this

Reply:
CSVReader was kind of a mess. I dont mind re-visiting trying it, originally, we wanted to just write out the blobs using with something like Table -> toBlob -> write(stringstream) -> write to HDF5. But that also was a mess. If we revisit I think trying the blob route would be best. She was able to get it work with a quick ghetto method than I was trying to de-tangle Blob writing with streams. Only spent a day on it, but I was getting a bunch of segfaults and stream errors for whatever reason.